lite/examples/sound_classification/ios/SoundClassification/GMLAudio/GMLAudio.h

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "GMLRingBuffer.h"
+#import "GMLAudioRecord.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** A wrapper class to store input audio used in on-device machine learning. */
+NS_SWIFT_NAME(Audio)
+@interface GMLAudio : NSObject
```

Comment:
After discussion with the TFLite engineers, we decided to use TFLAudioTensor rather than GMLAudio and we'll put them inside the ios/ folder.

Reply:
So I can use the error utils from TFLCommonUtils right? Also add the error codes to TFLCommon?

Reply:
Also should we rename GMLAudioRecord to TFLAudioRecord. Note that the Audio record is almost an independent module. It can be used by any library if need be.

Reply:
NVM, I am keeping it consistent to TFL for now since the audio formats used by both AudioTensor and AudioRecord are same. Later if needed, we can make it a separate library.

Reply:
> So I can use the error utils from TFLCommonUtils right? Also add the error codes to TFLCommon?

Yes that's correct.

> Also should we rename GMLAudioRecord to TFLAudioRecord. Note that the Audio record is almost an independent module. It can be used by any library if need be.

Yes we should have all classes named with TFL prefix as they are all part of TFLite Task Library now.

Reply:
Fixed